warp10/src/main/java/io/warp10/script/ext/token/TOKENDUMP.java

```diff
@@ -102,6 +104,7 @@ public Object apply(WarpScriptStack stack) throws WarpScriptException {
 
       AESKey = (byte[]) top;
 
+      customKeys = true;
```

Comment:
Thus `customKeys == multikey`. You could replace every use of `customKeys` by `multikey`.